.github/workflows/leancode_contracts_generator-publish.yml

```diff
@@ -20,7 +20,7 @@ jobs:
 
       - uses: dart-lang/setup-dart@v1
         with:
-          sdk: 2.16.1
+          sdk: 2.16.x
```

Comment:
wow, kool!

Reply:
Unfortunately not: 
<img width="409" alt="image" src="https://user-images.githubusercontent.com/29288116/178050556-c796bc59-7516-4a6f-8fe9-c5b46ca72484.png">


Most actions support version placeholders, apparently dart does not :( (something worth contributing!)

Reply:
https://github.com/dart-lang/setup-dart/issues/56